spec/controllers/tours_controller_spec.rb

```diff
@@ -1,5 +1,55 @@
 require 'rails_helper'
 
 RSpec.describe ToursController, type: :controller do
+  let(:user) { create :staff_user }
 
+  context "User authenticated" do
+    before(:each) do
+      authenticate_staff_user(user)
+    end
+
+    it 'creates a tour associated with the current user and their cultural center when provided valid data' do
+      post :create, params: {
+        tour: {
+          title: "Awesome Tour",
+          staff_user_id: user.id,
+          cultural_center_id: user.cultural_center.id
+        }
+      }
+      tour = Tour.all.last
```

Comment:
I don't really like using `last` in tests like this. Instead, you could say:

```ruby
let(:json) { JSON.parse(response.body) }
```

Then, in your test, you'd change:

```diff
-tour = Tour.all.last
+tour = Tour.find(json['id'])
```

The benefits are:
1. You know for certain that the tour you've just found is the one that was created in the test.
2. If the tour isn't found, the test will raise an error.